api_types.ts

```diff
@@ -883,6 +883,16 @@ export interface ExecutionContext {
    * Information about state of the current sync. Only populated if this is a sync table formula.
    */
   readonly sync?: Sync;
+
+  /**
+   * If this invocation is a part of a crawling execution, like in Coda Brain, then this ID will be provided
+   * to all invocations. That includes invocations of sync `execute` and `executeGetPermissions`, as well as
+   * dynamic table features like `listDynamicUrls`, `getSchema`, and `getName`.
```

Comment:
Ah is this the reason for not putting it inside of `sync`?

Reply:
Yup!